__tests__/app.test.js

```diff
@@ -0,0 +1,48 @@
+const app = require("../app");
+const request = require("supertest");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const {
+  topicData,
+  userData,
+  articleData,
+  commentData,
+} = require("../db/data/test-data/index");
+
+beforeEach(() => {
+  return seed({ topicData, userData, articleData, commentData });
+});
+
+afterAll(() => {
+  return db.end();
+});
+
+describe("GET /api/topics", () => {
+  test("get 200", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        const { topics } = body;
+
+        expect(topics).toBeInstanceOf(Array);
+        expect(topics).toHaveLength(3);
+        topics.forEach((topic) => {
+          expect(topic).toEqual(
+            expect.objectContaining({
+              slug: expect.any(String),
+              description: expect.any(String),
+            })
+          );
+        });
+      });
+  });
+  test("STATUS 404", () => {
+    return request(app)
+      .get("/api/no-topics")
+      .expect(404)
+      .then((err) => {
+        return err;
```

Comment:
In this test you should be checking for a specific `msg` , which should be a custom error message you are sending back to the client. Currently this is just checking that the code is `404` - which is what express is sending back by default